indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -56,23 +68,80 @@ def __table_args__(cls):
         index=True,
         nullable=False
     )
+    user_id = db.Column(
```

Comment:
I would name it `creator_id`, same way we have `judge_id`.
`Abstract.user_id` doesn't tell me anything, but `Abstract.creator_id` is self explanatory.
If there is a strong argument against this, I would at least add a comment above the column definition.